From ae27d5b4be42cd98b3db299d161a2e3ea77eb604 Mon Sep 17 00:00:00 2001
From: Evgeny Vereshchagin <evvers@ya.ru>
Date: Wed, 11 May 2022 22:32:32 +0000
Subject: [PATCH] tests: ignore dbus-broker-launcher

There are memory leaks there https://github.com/bus1/dbus-broker/issues/289
and it crashes from time to time
https://github.com/matusmarhefka/dfuzzer/issues/20#issuecomment-1114097840
so let's just skip it by analogy with dbus-daemon to avoid
reports that have nothing to do with systemd itself.

It's kind of a part of https://github.com/systemd/systemd/pull/22547

(cherry picked from commit d0880faa5dda495c7c77425697b82a94b4e68bf6)
Related: #2087652
---
 test/test-functions | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test/test-functions b/test/test-functions
index a299f5ff1f..7c37d05610 100644
--- a/test/test-functions
+++ b/test/test-functions
@@ -1328,6 +1328,7 @@ check_asan_reports() {
                  BEGIN {
                      %services_to_ignore = (
                          "dbus-daemon" => undef,
+                         "dbus-broker-launch" => undef,
                      );
                  }
                  print $2 if /\s(\S*)\[(\d+)\]:\s*SUMMARY:\s+\w+Sanitizer/ && !exists $services_to_ignore{$1}'
